test/jdk/java/net/vthread/HttpALot.java

```diff
@@ -21,7 +21,21 @@
  * questions.
  */
 
-/**
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.net.Proxy;
+import java.net.URL;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.atomic.AtomicInteger;
+import com.sun.net.httpserver.HttpServer;
+import jdk.test.lib.net.URIBuilder;
```

Comment:
Did you mean to move the imports? Just curious as I usually put the test description at the top to make it easy to find, I guess others prefer it after the imports.

Reply:
Hello Alan, I will undo that part. I usually prefer the test definition directly above the class declaration.

Reply:
I just happened to revert this part in the PR :) So I'll let it be before the imports, like it was before. No changes to the `@test` definition, except replacing `/**` with the recommended `/*`.